.github/workflows/release-suite-native-preview-update.yml

```diff
@@ -0,0 +1,49 @@
+name: "[Release] suite-native preview update"
```

Comment:
I'm not sure I like the name of this, mainly release, as it is more like a build test app rather than release.

Reply:
It is the same as it was with [Release] suite-native develop, but let's change it to build and keep release just for the public release.

Reply:
https://github.com/trezor/trezor-suite/pull/14530/commits/087a775cc2295874b616cd4d2ae8d79ead1f542a